tests/NSubstitute.Acceptance.Specs/NSubstitute.Acceptance.Specs.csproj

```
@@ -1,13 +1,13 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp2.0;netcoreapp1.1;net46;net45;net5.0</TargetFrameworks>
+    <TargetFrameworks>net46;net45;net5.0;net6.0</TargetFrameworks>
```

```suggestion
    <TargetFrameworks>netcoreapp2.0;netcoreapp1.1;net46;net45;net5.0;net6.0</TargetFrameworks>
```

Let's keep netcoreapp* as targets for tests.

---

I added netcoreapp2.1 there, having netcoreapp1.1 then probably needs nunit and other nuget package downgrades I tried with conditional itemgroups and couldnt make tests working in visual studio. netcoreapp1.1 has been out of support for over year

---

sounds good to me! thank you!